client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-definitions/setup/index.tsx

```diff
@@ -14,6 +15,19 @@ const getSetupFreeTask: TaskAction = ( task, flow, context ): Task => {
 	};
 };
 
+const getSetupBlog: TaskAction = ( task, flow, context ): Task => {
+	const { siteInfoQueryArgs } = context;
+
+	return {
+		...task,
+		actionDispatch: () => recordTaskClickTracksEvent( task, flow, context ),
```

Comment:
I have a concern around the fact that we're implementing what should be generic Tracks triggers in this task-specific logic. This should be somewhere central so we don't have cases where it's accidentally left off for some tasks, or where we make some change that doesn't get rolled out to all locations. Can this be moved into `getTaskDefinition()`?

Reply:
I totally agree but I prefer finish this migration first and then iterate around it. My idea is to have a full vision about the problem first.